content/_data/indexpage/carousel.yml

```
@@ -1,3 +1,14 @@
+- :href: https://contributors.jenkins.io/
+  :title: Meet the driving forces
+  :intro: behind Jenkins 
+    as we showcase the top contributors shaping the
+    future of continuous integration and delivery
```

```suggestion
    future of continuous integration and delivery.
```
adding full stop for end of sentence